__tests__/app.test.js

```diff
@@ -13,28 +13,71 @@ afterAll(() => {
 });
 
 describe("app", () => {
-  describe("/api", () => {
-    test("200: GET responds with a message of connection", () => {
-      return request(app)
-        .get("/api")
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Connected");
-        });
-    });
-  });
   describe("/api/categories", () => {
     test("200: responds with an array of categories", () => {
       return request(app)
         .get("/api/categories")
         .expect(200)
         .then((body) => {
           const categories = body.body;
```

Comment:
So we don't have to do .body.body to access the data - we can instead destructure when sending data on the controllers - ie .send({reviews}) and like so on the test side -> .then(({body}